.travis.yml

```diff
@@ -7,22 +8,72 @@ python:
   - "3.4"
   - "3.5"
   - "3.6"
-  - "pypy"
+  - "3.7"
+  - "pypy2.7-6.0"
+  - "pypy3.5-6.0"
 
-matrix:
+env:
+  global:
+    LIBGIT2: ~/libgit2/_install/
+    LD_LIBRARY_PATH: ${LIBGIT2}/lib:${LD_LIBRARY_PATH}
+    LIBGIT2_VERSION: "0.27"
+
+
+jobs:
   include:
-    - python: 3.7
-      dist: xenial
-      sudo: true
+  - stage: &deploy_stage Publish to PyPI (runs only for tagged commits)
+    name: >-
+      OS-specific manylinux1 wheels
+    python: "3.7"
+    services:
+    - docker
+    install: skip
+    script: &docker-script
+    - travis/build-all-manylinux1-wheels.sh "pygit2" "${LIBGIT2_VERSION}"
+    before_deploy:
+    - ls dist
+    - pip install twine
+    - twine check dist/*
+    deploy:
+      provider: pypi
+      # `skip-cleanup: true` is required to preserve binary wheels, built
+      # inside of manylinux1 docker container during `script` step above.
+      skip-cleanup: true
+      # `skip-existing: true` is required to skip uploading dists, already
+      # present in PYPI instead of failing the whole process.
+      # This happens when other CI (AppVeyor etc.) has already uploaded
+      # the very same dist (usually sdist).
+      skip-existing: true
+      user: jdavid
+      password:
+        # Encrypted with `travis encrypt -r aio-libs/aiohttp --api-endpoint 'https://api.travis-ci.com/'`:
+        secure: >-
+          TODO: REPLACE THIS WITH AN ENCRYPTED PASSWORD
```

Comment:
@jdavid please generate the encrypted string using `travis encrypt -r libgit2/pygit2` command.

And then you can use this page to update in via UI right in this PR: https://github.com/webknjaz/pygit2/edit/feature/manylinux1-binary-wheels/.travis.yml?pr=%2Flibgit2%2Fpygit2%2Fpull%2F869